zerver/lib/event_schema.py

```diff
@@ -1542,3 +1542,26 @@ def check_user_status(var_name: str, event: Dict[str, object], fields: Set[str])
     _check_user_status(var_name, event)
 
     assert set(event.keys()) == {"id", "type", "user_id"} | fields
+
+
+stream_user_group_access_object_type = DictType(
+    required_keys=[("id", int), ("stream_id", int), ("group_id", int)]
+)
```

Comment:
I think we `# force vertical` keys like this in `event_schema.py`.

Reply:
It is auto formatted by black, it depends on how much text is there in a line.

Reply:
If you put a comma after the last element of `required_keys`, I think black won't reformat it.
```python
stream_user_group_access_object_type = DictType(
    required_keys=[
        ("id", int),
        ("stream_id", int),
        ("group_id", int),
    ]
)
```
Look like this is followed throughout `event_schema.py`, probably for readability.